New package: HmtGutenberg v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: HmtGutenberg
- Repository: https://github.com/homermultitext/HmtGutenberg.jl
- Created by: @neelsmith
- Version: v0.1.0
- Commit: 5c3b3f00e9696eaf8629421c85909e4556277903
- Git reference: main
- Description: Generate plain-text editions of texts in the HMT archive in as many formats as you please.
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
Initial release, with docs on quarto.pub.
`````
<!-- END RELEASE NOTES -->

goerz commented:
Thank you for submitting your package! However, please make sure to add some documentation before registering. [At the very least](https://discourse.julialang.org/t/when-does-it-make-sense-to-post-a-new-package-to-the-julia-repository/108701/4), that would be a small usage example in the README

JuliaHubRegistrator committed:
New package: HmtGutenberg v0.1.0

UUID: 09780231-1698-4933-9d9d-4e01e0abea36
Repo: https://github.com/homermultitext/HmtGutenberg.jl.git
Tree: 1557aaa622fe2951f8a57284706cbda7102386b0

Registrator tree SHA: 191228b6dd8b9d0e2965ae3e705fe54c51dcfee8
